code/html/custom.js

```diff
@@ -1025,6 +1030,10 @@ function initRelayConfig(data) {
         $("select[name='relayBoot']", line).val(data.boot[i]);
         $("select[name='relayPulse']", line).val(data.pulse[i]);
         $("input[name='relayTime']", line).val(data.pulse_time[i]);
+        $("input[name='relayLastSch']", line).prop('checked', data.sch_last[i]);
+        $("input[name='relayLastSch']", line).attr("id", "relayLastSch" + i);
+        $("input[name='relayLastSch']", line).attr("name", "relayLastSch" + i);
+        $("input[name='relayLastSch" + i + "']", line).next().attr("for","relayLastSch" + (i));
```

Comment:
Why do this part? Every other relay element is unindexed and depends on parent elem ordering. Could this be just the `prop('checked', ...)` and remove the rest? Then the data gathering does not need this exceptional handling

Reply:
This is what i tried to do at the beggining but what happened is that the `createCheckboxes` function was called before the `initRelayConfig` got initialized, this caused the checkbox the not function correctly.
The same issue already happens in the schedule template and you deal with it in the same way
`$(line).find("input[name='schEnabled']").prop("id", "schEnabled" + (numSchedules + 1))
        .next().prop("for", "schEnabled" + (numSchedules + 1));`

Reply:
Yikes, I see. Is createCheckboxes the culprit though? [After little refresher about how checkboxes were implemented here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#attr-for), label for=... needs unique id as it seems.
Sorry about that.

For specifics:
```
<html>
    <body>
        <div class="block" id="block-a">
            <input name="input-name" id="input-name" type="checkbox"></input>
            <label for="input-name">Name a</label>
        </div>
        <div class="block" id="block-b">
            <input name="input-name" id="input-name" type="checkbox"></input>
            <label for="input-name">Name b</label>
        </div>
        <div class="block" id="block-c">
            <input name="input-name" id="input-name-c" type="checkbox"></input>
            <label for="input-name-c">Name c</label>
        </div>
        <div class="block" id="block-d">
            <input name="input-name" id="input-name-d" type="checkbox"></input>
            <label for="input-name-d">Name d</label>
        </div>
    </body>
</html>
```
Clicking Name B will not select the closest one, inside block-b, but the one from block-a
Unique ids work as intended and this is what happening for input+label combo